src/github.com/couchbase/sync_gateway/db/change_cache.go

```diff
@@ -13,8 +15,10 @@ import (
 	"github.com/couchbase/sync_gateway/channels"
 )
 
-var MaxChannelLogPendingCount = 10000               // Max number of waiting sequences
-var MaxChannelLogPendingWaitTime = 60 * time.Second // Max time we'll wait for a missing sequence
+var DefaultMaxChannelLogPendingCount = 10000              // Max number of waiting sequences
+var DefaultMaxChannelLogPendingWaitTime = 5 * time.Second // Max time we'll wait for a pending sequence before sending to missed queue
+
+var DefaultMaxChannelLogSkippedWaitTime = 60 * time.Minute // Max time we'll wait for an entry in the missing before purging
```

Comment:
I changed these from variables -> constants since they appeared to be read only (and pushed a new commit to the branch).  Let me know if there is a specific reason they need to be variables.